src/composables/useBalanceCalculator.js

```diff
@@ -80,12 +89,9 @@ export function updateBilanz(entry) {
   }
 
   bilanz = anyErrors ? [] : calculateBilanz();
-  return [bilanz, errors];
+  return { bilanz, errors };
```

Comment:
Hier eignet sich ein Object besser als ein Array, siehe auch die geänderte Verwendung in `TheFertilizerBalance.vue`